build.sbt

```diff
@@ -219,7 +201,8 @@ lazy val ratatoolScalacheck = project
     libraryDependencies ++= Seq(
       "org.apache.avro" % "avro" % avroVersion,
       "org.scalacheck" %% "scalacheck" % scalaCheckVersion,
-      "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion
+      "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion,
+      "org.apache.beam" % "beam-sdks-java-core" % beamVersion
```

Comment:
It doesn't seem right that scalacheck generators depend on beam. Why is it needed?

Reply:
Solely to be able to use `CoderUtils` from Beam. Otherwise to do any operations in Scio or Beam results in errors like this 
```org.apache.avro.UnresolvedUnionException: Not in union ["null",{"type":"map","values":"int"}]: {tfRpuigtrpwsHlclcgiuoRhund=0, igplyAuvDfxiwiuPdgnsudlalyasbh=-1561596481}
    at org.apache.avro.generic.GenericData.resolveUnion(GenericData.java:740)
    at org.apache.avro.generic.GenericDatumWriter.resolveUnion(GenericDatumWriter.java:205)
    at org.apache.avro.generic.GenericDatumWriter.writeWithoutConversion(GenericDatumWriter.java:123)
    at org.apache.avro.specific.SpecificDatumWriter.writeField(SpecificDatumWriter.java:87)
    at org.apache.avro.reflect.ReflectDatumWriter.writeField(ReflectDatumWriter.java:191)
    at org.apache.avro.generic.GenericDatumWriter.writeRecord(GenericDatumWriter.java:156)
    at org.apache.avro.generic.GenericDatumWriter.writeWithoutConversion(GenericDatumWriter.java:118)
    at org.apache.avro.specific.SpecificDatumWriter.writeField(SpecificDatumWriter.java:87)
    at org.apache.avro.reflect.ReflectDatumWriter.writeField(ReflectDatumWriter.java:191)
    at org.apache.avro.generic.GenericDatumWriter.writeRecord(GenericDatumWriter.java:156)
    at org.apache.avro.generic.GenericDatumWriter.writeWithoutConversion(GenericDatumWriter.java:118)
    at org.apache.avro.generic.GenericDatumWriter.write(GenericDatumWriter.java:75)
    at org.apache.avro.reflect.ReflectDatumWriter.write(ReflectDatumWriter.java:159)
    at org.apache.avro.generic.GenericDatumWriter.write(GenericDatumWriter.java:62)
    at org.apache.beam.sdk.coders.AvroCoder.encode(AvroCoder.java:308)
    at org.apache.beam.sdk.coders.Coder.encode(Coder.java:135)
    at org.apache.beam.sdk.util.CoderUtils.encodeToSafeStream(CoderUtils.java:93)
    at org.apache.beam.sdk.util.CoderUtils.encodeToByteArray(CoderUtils.java:77)
    at org.apache.beam.sdk.util.CoderUtils.clone(CoderUtils.java:156)```

`CoderUtils.clone` is an operation most tests do and this avoids it and still works using direct avro lib functions. Do you have another suggestion?

Reply:
I think AVRO has all necessary APIs, you can refer to `AvroCoder` implementation for it.

Reply:
`CoderUtils` doesn't replicate scio coding behavior because there it's possible to register serializer, that's what we actually do to workaround `AVRO-1891`.

Reply:
I have mixed feelings about this. Requires context/knowledge from the end user, current approach will work out of the box. I will split this into a separate issue that can be looked into outside the scope of this PR.

Reply:
https://github.com/spotify/ratatool/issues/46